OpenTerm/Controller/ViewController.swift

```diff
@@ -30,6 +30,10 @@ extension String {
 
 }
 
+let executor = CommandExecutor()
+
+var activeVC: ViewController!
```

Comment:
Both of the above lines conflict with the Tabs change, which will introduce multiple `ViewController` instances, and moves `executor` to the `TerminalView`.

To get the visible view controller, there is an API on the `TerminalTabViewController` called `visibleViewController`